tools/binlog-slave-client.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Binlog Slave Client User Guide
+summary: Learn how to use Binlog Slave Client.
```

Comment:
The information conveyed in this sentence is a little bit too less. The summary sentence is intended to benefit SEO. We need to think how to convey useful info concisely. Perhaps we can change it to:

Use Binglog Slave Client to parse the binlog data and output the data in a specific format to Kafka.